tools/cldr-apps/js/src/esm/cldrVote.mjs

```diff
@@ -342,20 +342,8 @@ function showProposedItem(inTd, tr, theRow, value, tests, json) {
       ];
     }
 
-    var input = $(inTd).closest("tr").find(".input-add");
-    if (input) {
-      input.closest(".form-group").addClass("has-error");
-      input
-        .popover("destroy")
-        .popover({
-          placement: "bottom",
-          html: true,
-          content: cldrSurvey.testsToHtml(tests),
-          trigger: "hover",
-        })
-        .popover("show");
-      if (tr.myProposal) tr.myProposal.style.display = "none";
-    }
+    const description = cldrSurvey.testsToHtml(tests);
+    cldrNotify.openWithHtml("Response to voting", description);
```

Comment:
It'd be cleaner here (and avoid HTML injection issues?) by just opening a Vue form here that takes `{tests}` as a property and then displays the test in a structured way.

Reply:
yes... I need to study the server response more to see what it can contain... for this PR I didn't want to go too far all at once